app/publisher/publisher.py

```diff
@@ -0,0 +1,37 @@
+import google
```

Comment:
We should define an abstract publisher base class to make it clear how a publisher should be implemented.

Reply:
Done but had to remove the `PublisherClient` inheritance due to miss matching method signatures.